apps/badass/src/components/icons.tsx

```diff
@@ -0,0 +1,62 @@
8-0.341-17.493 2.645-34.901 11.52-50.602 2.56-4.523 3.925-9.216 4.096-14.592 0.341-9.899 4.181-18.518 11.861-25.088 1.451-1.28 2.987-2.56 2.902-4.779-0.171-3.925 1.877-6.997 3.925-9.899 4.864-6.826 9.387-14.165 11.179-22.186 1.792-8.107 6.058-13.824 12.117-18.518 3.499-2.73 7.083-4.949 10.325-8.192 5.462-5.546 8.79-12.373 13.398-18.346 0.597-0.854 0.682-2.219 1.45-2.731 10.411-7.424 12.032-20.053 18.176-30.037 6.742-11.008 9.643-24.832 22.955-31.574 6.912-3.498 11.008-10.837 15.616-16.81 8.875-11.435 21.76-18.688 29.952-30.72 4.096-5.974 7.851-13.398 15.019-16.043 9.898-3.584 14.933-13.653 24.576-17.152 1.621-0.597 2.645-2.133 3.328-3.755 4.864-11.349 14.677-14.421 25.77-14.848 2.134 0 3.584-0.341 4.864-1.962 6.656-8.107 15.19-11.776 25.686-11.35 7.424 0.342 14.933 0.342 22.357 0 5.888-0.256 10.325 2.987 14.421 5.974 3.072 2.218 4.95 6.4 4.096 10.922-1.365 6.827-2.73 13.654-3.84 20.48-0.341 1.792-0.682 3.499-2.048 4.523-9.386 6.656-9.984 19.115-17.92 27.051-3.242 3.242-5.632 7.168-10.069 8.96-8.704 3.413-13.824 11.52-21.333 16.213-7.766 4.864-11.179 12.032-15.446 19.2-2.901 4.779-6.997 8.533-12.629 10.069-4.096 1.195-6.485 3.584-7.339 7.851-1.792 8.448-7.338 13.909-14.933 17.408-2.389 1.109-3.328 3.072-4.523 5.12-5.12 8.277-11.264 15.701-17.664 22.955-0.426 0.512-0.853 1.109-1.45 1.194-11.179 2.902-13.142 13.142-18.262 21.334-5.376 8.704-7.936 19.882-16.384 25.941s-11.776 13.995-14.592 22.955c-0.341 1.024-0.938 1.962-1.621 2.73-6.741 8.022-9.216 17.92-11.435 27.734-0.682 2.901-3.413 6.4 0.256 8.789 3.584 2.304 9.046 2.389 11.435-0.085 6.4-6.571 14.677-6.23 22.272-8.875 10.752-3.755 21.419-3.328 32.085-2.901 10.155 0.341 20.31 1.024 30.379 2.986 6.229 1.195 12.8-0.938 19.456 1.451 3.669 1.28 6.4 3.499 9.045 5.547 5.462 4.096 12.715 4.352 17.664 9.301 9.472 9.557 20.822 17.323 28.758 28.416 5.205 7.253 13.397 11.435 18.432 18.773 0.512 0.683 1.45 1.195 1.621 1.963 1.877 8.619 6.571 15.787 10.837 23.296 2.56 4.437 3.414 10.155 5.888 15.104 1.792 3.584 3.072 7.509 2.304 13.483l-0.17-0.342zM719.043 731.825c4.95 1.877 7.083 0.171 6.827-5.632-0.171-3.84 1.621-7.936 4.181-10.411 4.011-3.754 3.755-8.362 3.926-12.885 0.17-2.731 0.17-5.461 0.426-8.192 0.683-8.021 3.499-15.787 4.096-23.467 0.939-11.264 0-22.784 2.987-33.962 0.341-1.195 0-2.816-1.621-2.902-1.024-0.085-2.048 0.854-2.219 1.963-0.853 4.949-2.219 9.899-2.304 14.848-0.256 11.264-6.229 19.115-16.725 20.651-3.67 0.512-4.352 2.645-4.608 5.717-0.427 5.717-1.707 11.179-2.987 16.725-2.304 10.752-5.973 21.248-6.571 32.342-0.256 4.266 1.536 6.997 6.571 6.058 2.475-0.512 5.035-0.512 8.021-0.853zM683.203 665.18c-0.17-0.512 0.598-2.731-1.28-3.499s-3.157 0.939-4.266 2.048c-3.328 3.328-4.864 7.68-4.438 12.117 0.598 6.656-1.621 12.203-5.632 16.896-2.645 3.072-4.693 6.315-6.229 10.070-1.109 2.645-4.352 4.949-2.048 8.021 2.987 4.011 7.851 5.547 12.288 7.424 1.109 0.427 2.133-0.427 2.389-1.536 2.304-11.093 5.462-22.016 5.462-33.365 0-4.182 0.341-8.022 2.389-11.606 0.939-1.706 1.877-3.328 1.451-6.4l-0.086-0.17z" />
```

Comment:
@DrShpongle I suggest importing these complicated SVGs from public folder and using Image component for rendering.

Reply:
I've realized nice hover effect, it will be awkward to do it with images, but if you think icons should gone I'll substitude it with img.
![CleanShot 2023-06-30 at 16 56 46](https://github.com/skillrecordings/products/assets/1519448/24b9ab26-fe2e-4b39-90be-d84687e40c07)